src/app/form/form.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="wrapper">
+    <div class="price-fill-form">
+        <h2>{{currentApartment.price}} per week</h2>
+        <h4> Fill out the form below to speak with us regarding availability. </h4>
+    </div>
+
+    <div class="wraper-form">
+        <div class="content-form">
+            <form [formGroup]="currentStudent" (ngSubmit)="submit()">
+                <div class="form-group">
+                    <label class="label">
+                      <span>Full Name</span>
+                      <em class="required-star">*</em>
+                    </label>
+                    <div class="content-name">
+                        <span class="first-name">
+                      <input type="text" class="form-control" name="first-name" formControlName="studentFirstName">
+                      <label class="form-sub-info-text">First</label>
+                    </span>
+                        <span class="last-name">
+                      <input type="text" class="form-control" name="last-name" formControlName="studentLastName">
+                      <label class="form-sub-info-text">Last</label>
+                    </span>
+                    </div>
+                    <div class="clearfix"></div>
+                    <div class="warning" *ngIf="currentStudent.controls['studentFirstName'].invalid && currentStudent.controls['studentFirstName'].touched || 
+                    currentStudent.controls['studentLastName'].invalid && currentStudent.controls['studentLastName'].touched">
+                        Enter a value for this field.
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+                    <span>Phone Number</span>
+                    <em class="required-star">*</em>
+                  </label>
+                    <input type="tel" class="form-control" name="phone" formControlName="studentPhone">
+                    <div class="warning" *ngIf="currentStudent.controls['studentPhone'].invalid && currentStudent.controls['studentPhone'].touched">
+                        Enter a number for this field like this (+380) 63-889-11-89.
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+                  <span>Email Address</span>
+                </label>
+                    <input type="email" class="form-control" name="email" formControlName="studentEmail">
+                    <div class="warning" *ngIf="currentStudent.controls['studentEmail'].invalid && currentStudent.controls['studentEmail'].touched">
+                        Enter a valid email address. (eg: yourname@domain.com)
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+                <span>University / College of attendance</span>
+              </label>
```

Comment:
formatting

Reply:
I can't to align this label. When I'm selecting a part of the code which I wish to align, and I'm pressing shortcut Shift+Alt+F (for align html code on Windows), nothing changes.

Reply:
align this code by your hand
why <span> bellow <label>